testsuite/tests/tool-ocamldoc-2/inline_records.reference

```diff
@@ -50,14 +76,114 @@ A simple record type for reference
 
 \label{TYPInline-underscorerecords.t}\begin{ocamldoccode}
 type t =
-  | C of {\char123}
-   lbl : int;
-   more : int list;
-{\char125}
+
 \end{ocamldoccode}
```

Comment:
It was an error due to some problem with execution order within `odoc_latex`. The field documentation was originally preprocessed but only in text mode (i.e. without some of the fancy latex feature). However, I found the time to rewrite the code to support a full latex mode for field documentations.